### DISCUSSION TITLE
Duplicated react-joyride elements on the body

### DISCUSSION THREAD

**Discussion by ayozebarrera** [gilbarbara/react-joyride#677] [Q&A] [answered]
Hey there!

I'm using react-joyride in a functional component and everything was working fine until I saw this on my body:

![Captura de pantalla 2021-04-14 a las 9 25 38](https://user-images.githubusercontent.com/6180188/114678670-6337dc00-9d03-11eb-95c2-ec3a4e9d7503.png)

I debugged my component and it is not re-rendering... react-joyride either but when I set joyride to run, I noticed on the dom is executing like twice but debugging react-joyride tells me not:

![Captura de pantalla 2021-04-14 a las 9 29 16](https://user-images.githubusercontent.com/6180188/114679305-f40eb780-9d03-11eb-9cbf-5ca73e28d89c.png)

If anyone has a clue of what is happening here, please tell me... I'll appreciate it

Thanks!

**Comment by if2527** [comment]
I have the same  when using react-joyride in strict mode. Don't know how to fix it.

**Reply by ayozebarrera** [reply, marked as answer]
Same here... I don't know if this a library bug but in the demo this doesn't happens.
